docs/pages/blog/making-customizable-components.md

```diff
@@ -0,0 +1,289 @@
+---
+title: 'Making customizable components'
+description: The use case of of data grid
+date: 2022-06-23T00:00:00.000Z
+authors: ['alexfauquette']
+tags: ['MUI X']
+---
+
+At MUI, our goal is to empower as many people as possible to build great UIs.
+To do so, we build React components and documentation used by hundreds of thousands of developers on everything from minor side projects to massive company websites.
+
+This variety of users presents a dilemma: hobbyists working on side projects want fully built components that work right out of the box, so they can focus on the application logic; many larger companies, by contrast, want to be able to fully customize components to respect their brand design.
+Managing these contradictory needs becomes harder as component complexity increases.
+
+This article reviews several different approaches to customizing components, as well as the various tradeoffs associated with each method.
+Along the way, we’ll explore how these tradeoffs ultimately led to the solution that we’ve settled on for customizing MUI components.
+
+## Style modification
+
+If you already use a styling library, don’t hesitate to skip this part.
+See you back in Logic modification section
```

Comment:
Should we keep that.

The CSS section is a reminder about what works well. 

- Advanced users already know that part of customization very well, so they probably will want to skip this section
- Junior dev could be interested by this short introduction about style modification